PCIe 6.0 Packet Generator and Exerciser/PCIe 6.0 Packet Generator and Exerciser/utils/nonHeaderBase.h

```diff
@@ -0,0 +1,80 @@
+#pragma once
+// Abstact class
+class NonHeaderBase {
+public:
+	int requestID;
```

Comment:
Please move the size attribute to be in the parent class and just set by the constructors of the concrete classes. Also, please rename it to headerSizeInBytes for clarity